libs/functions/get_can_projection.py

```diff
@@ -15,9 +15,9 @@ def _get_interventions_for_state(state):
 
 
 def _get_intervention(intervention, state):
-    if intervention == Intervention.CURRENT.value:
+    if intervention == Intervention.CURRENT:
         state_intervention = _get_interventions_for_state(state)
-        return Intervention.from_str(state_intervention).value
+        return Intervention.from_str(state_intervention)
```

Comment:
FYI: I don't know python well enough to understand these 2 changes, but I assume they're good!